libpod/container.go

```diff
@@ -194,6 +194,8 @@ type ContainerConfig struct {
 	CreatedTime time.Time `json:"createdTime"`
 	// Cgroup parent of the container
 	CgroupParent string `json:"cgroupParent"`
+	// NoNewPrivs indicates whether the container will run with NoNewPrivs
+	NoNewPrivs bool `json:"noNewPrivs"`
```

Comment:
Can you throw this a bit higher up in the ContainerConfig struct, with the other security-related options (SELinux labels and user)?